web/src/components/Button/index.tsx

```diff
@@ -14,20 +12,23 @@ export interface ButtonProps {
   size: string
 }
 
-const Button: React.FC<ButtonProps & WithStylesProps<typeof theme>> = ({
-  onClick,
-  disabled,
-  children,
-  classes,
-}) => (
-  <button
-    className={classes.button}
-    disabled={disabled}
-    type="button"
-    onClick={onClick}
-  >
-    {children}
-  </button>
-)
+const useStyles = createUseStyles(styles)
```

Comment:
I would move `createUseStyles` into the styles file directly!

Reply:
Just updated PR. Not sure how to name styles import - `import styles` or `import useStyles`. Used `useStyles` as it's function now